docs/daemon/setup/network.mdx

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 2
+sidebar_label: "Network "
+id: daemon-network-setup
+title: Network Setup
+description: Running Kwil Network
+slug: /daemon/network/setup
+---
+
+This guide provides a step-by-step instructions on preparing a setup to deploy a kwild network with N validator nodes and M non-validator nodes
+
+## Configuration
+kwil-admin setup can be used to generate configuration required for the N Validator + M Non-Validator nodes.
+
+For example, Configuration required to deploy a network with N=2 Validator nodes and M=1 Non-Validator node can be generated as below.
+
+If the ip addresses of these nodes are consecutive, you can instead use the starting-ip-address flag as follows:
+
+```bash
+kwil-admin setup testnet -v 2 -n 1 -o "path/to/output/dir" --starting-ip '192.168.10.10'
+```
+
+If the IP addresses are not consecutive, provide explicit ip addresses using --hostname as shown in the above command. 
+```bash
+kwil-admin setup testnet -v 2 -n 1 -o "path/to/output/dir" --hostname '192.168.10.10' --hostname '192.168.10.20' --hostname '192.168.10.30'
+```
+
+
+```bash title="Example Run"
+# Generate configuration required for a testnet corresponding 2 Validator nodes and 1 NonValidator node with IP addresses starting from '192.168.10.10' at location ~/.kwild
+
+$ kwil-admin setup testnet -v 2 -n 1 -o "~/.kwild" --starting-ip '192.168.10.10'
+Successfully initialized 3 node directories: ~/.kwild
+
+# Running the above command generates 3 directories node0, node1, node2 under ~/.kwild. 
+# Each node directories includes corresponding genesis.json, config.toml, private_key configuration files as shown below
```

Comment:
We have to manually wrap text inside code fences for it to be readable when rendered.

There is lots of horizontal scroll on this and the previous ones:

![image](https://github.com/kwilteam/docs/assets/140431406/9791dba7-be12-4dcf-b14a-70dc18d81d8d)

Wrapping comments is straightforward.
For long commands we can wrap like this:

```sh
kwil-admin setup testnet -v 2 -n 1 -o "path/to/output/dir" --hostname '192.168.10.10' \
  --hostname '192.168.10.20' --hostname '192.168.10.30'
```